pkg/services/ngalert/api/tooling/definitions/provisioning.go

```diff
@@ -7,9 +7,10 @@ type AlertingFileExport struct {
 	Groups        []AlertRuleGroupExport     `json:"groups,omitempty" yaml:"groups,omitempty"`
 	ContactPoints []ContactPointExport       `json:"contactPoints,omitempty" yaml:"contactPoints,omitempty"`
 	Policies      []NotificationPolicyExport `json:"policies,omitempty" yaml:"policies,omitempty"`
+	MuteTimings   []MuteTimeIntervalExport   `json:"muteTimes,omitempty" yaml:"muteTimes,omitempty"`
 }
 
-// swagger:parameters RouteGetAlertRuleGroupExport RouteGetAlertRuleExport RouteGetContactpointsExport RouteGetContactpointExport RoutePostRulesGroupForExport
+// swagger:parameters RouteGetAlertRuleGroupExport RouteGetAlertRuleExport RouteGetContactpointsExport RouteGetContactpointExport RoutePostRulesGroupForExport RouteExportMuteTimings RouteExportMuteTiming
```

Comment:
Unfortunately, no because in that case, the codegen will put the export route after `/{name}` path and it won't be accessible.

Reply:
Could it be singular `RouteGetMuteTimingExport`? It looks like contactpoints follows the same pattern "contactpoints" for multiple and "contactpoint" for the `/{uid}` handler.

Reply:
I can rename `RouteExportMuteTiming` to `RouteGetMuteTimingExport` and it won't break anything. But to be consistent I will have to rename `RouteExportMuteTimings` to `RouteGetMuteTimingsExport`, and this will break because route `RouteGetMuteTiming` (path /api/v1/provisioning/mute-timings/{name}) will end up before the export path in https://github.com/grafana/grafana/blob/bec5995ef99404257309dc93457291cd9c3a8c01/pkg/services/ngalert/api/generated_base_api_provisioning.go#L396-L407